aptos-move/framework/aptos-framework/sources/GovernanceProposal.move

```diff
@@ -0,0 +1,18 @@
+/// Define the GovernanceProposal that will be used as part of on-chain governance by AptosGovernance.
+///
+/// This is separate from the AptosGovernance module to avoid circular dependency between AptosGovernance and Stake.
```

Comment:
I'm not seeing this...

Reply:
It's because the setters in Stake need a GovernanceProposal for proof, while AptosGovernance uses GovernanceProposal and depends on Stake for voting power calculation. Hence if GovernanceProposal is not in a separate module, there'd be a circular dependency between AptosGovernance and Stake.